website/docs/terms/reverse-etl.md

```diff
@@ -1,10 +1,15 @@
 ---
 id: reverse-etl
 title: Reverse ETL
+description:  Reverse ETL is the process of getting your transformed data stored in your data warehouse to end business platforms, such as sales CRMs and ad platforms
 displayText: reverse ETL
 hoverSnippet: Reverse ETL is the process of getting your transformed data stored in your data warehouse to end business platforms, such as sales CRMs and ad platforms. 
 ---
 
+<head>
+    <title>Reverse ETL, demystified: What it is in plain english</title>
```

Comment:
```suggestion
    <title>Reverse ETL, demystified: What it is in plain english?</title>
```

Reply:
"What it is in plain english" is a statement, and does require a question mark

Reply:
yeah my b, read this wrong